### PR TITLE
Pan zoom

### PR DESCRIPTION
**Pull request by octavianCraciun**
In flot.navigate when interaction with the plot is in progress, the auto-scale will be disabled.
The auto-scale will be recovered at double-click.

PS: changes only in jquerry.flot.navigate.js

### CODE REVIEW

File: jquery.flot.js [hunk 1]
```diff
@@ -1232,8 +1232,8 @@ Licensed under the MIT license.
 
         function setRange(axis) {
             var opts = axis.options,
-                min,
-                max,
+                min = axis.min || opts.min,
```
Comment (cipix2000) [inline jquery.flot.js]: are you sure about this ? it triggers also for axis.min ===0

File: jquery.flot.js [hunk 2]
```diff
@@ -1242,16 +1242,16 @@ Licensed under the MIT license.
                     max = +(opts.max != null ? opts.max : axis.datamax);
                     break;
                 case "loose":
-                    if(axis.datamin != null && axis.datamax != null) {
-                        min = axis.datamin;
-                        max = axis.datamax;
+                    if(axis.datamin != null || axis.datamax != null) {
+                        min = isNaN(axis.datamin) ? opts.min : axis.datamin;
```
Comment (cipix2000) [inline jquery.flot.js]: is the axis.datamin ever expected to be NaN ?

File: jquery.flot.js [hunk 3]
```diff
@@ -1278,7 +1278,7 @@ Licensed under the MIT license.
             }
 
             // grow loose or grow exact
-            if(opts.autoscale !== "none" && opts.growOnly === true) {
+            if(opts.autoscale !== "none" && opts.autoscale !== "window" && opts.growOnly === true) {
```
Comment (cipix2000) [inline jquery.flot.js]: 'window' mode of autoscale isnot yet in

File: jquery.flot.js (hunk 2 above)
Comment (octavianCraciun) [inline jquery.flot.js]: when no data is available

File: jquery.flot.js (hunk 1 above)
Comment (octavianCraciun) [inline jquery.flot.js]: is only if autoscale is wrong set. it is overwritten later

File: jquery.flot.js (hunk 2 above)
Comment (octavianCraciun) [inline jquery.flot.js]: actually is null when no data is available.

**Pull request closed without merging**